### PR TITLE
PMM-4997: updated swagger-gen files sources json

### PR DESCRIPTION
**Pull request by lunaticusgreen**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "build": "ng build qan-app  --base-href '/qan/' --aot --output-path 'dist/qan-app' --prod --progress --output-hashing=all",
     "pack": "cd dist/; tar -czf qan-app-`date +%Y%m%dT%H%m%S`.tar.gz qan-app",
     "clean": "rm -r dist/*",
-    "pmm-api-gen": "ng-swagger-gen"
+    "pmm-api-gen": " ng-swagger-gen -c ./swagger-gen-config/ng-swagger-public.json && ng-swagger-gen -c ./swagger-gen-config/ng-swagger-public.json"
```
Comment (AlekSi) [inline package.json]: "public" is used twice.

Comment (AlekSi) [reply package.json]: There is also an extra space: `" n`

File: package.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
...
   },
```
Comment (askomorokhov) [inline package.json]: Twice `*-public.json`? Looks like a typo? :)

Comment (lunaticusgreen) [reply package.json]: Yep, copy-paste error

**Pull request merged**